api/src/pcapi/alembic/versions/20220922T123207_6858182908fe_delete_enable_phone_in_stepper_flag.py

```diff
@@ -0,0 +1,24 @@
+"""delete_enable_phone_in_stepper_flag
```

Comment:
Cette docstring est affichée quand on lance `alembic history`, il est utile de la rendre plus claire : 

```suggestion
"""Delete ENABLE_PHONE_VALIDATION_IN_STEPPER feature flag.
```

Reply:
ça marche ! Je modifie